src/config.h

```diff
@@ -1,5 +1,7 @@
-/* src/config.h.  Generated from config.h.in by configure.  */
+#include <xapian.h>
+
+/* Xapian::RangeProcesor (and subclasses) was introduced in 1.3.6. Therefore, with Xapian versions before 1.3.6, RXapian uses Xapian::ValueRangeProcessor (and subclasses) for supporting range queries. */
 #if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
-#define XAPIAN_RANGE_PROCESSOR_SUPPORT
+#define XAPIAN_RANGE_PROCESSOR_SUPPORT && XAPIAN_OP_MAX_SUPPORT
```

Comment:
Where can I find since which version it was supported?

On Fri, Aug 12, 2016 at 1:24 AM, James Aylett <notifications@github.com>
wrote:

> In src/config.h
> <https://github.com/amandaJayanetti/RXapian/pull/25#discussion_r74491369>:
>
> >  #if XAPIAN_MAJOR_VERSION >= 2 || XAPIAN_MINOR_VERSION >=3 || (XAPIAN_MINOR_VERSION == 3 && XAPIAN_REVISION >= 6)
> > -#define XAPIAN_RANGE_PROCESSOR_SUPPORT
> > +#define XAPIAN_RANGE_PROCESSOR_SUPPORT && XAPIAN_OP_MAX_SUPPORT
>
> That won't do what you think, and additionally OP_MAX is supported by
> earlier versions of Xapian, so you should test for it separately.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/25/files/72fb99ec5690760ac99cc99f918eccadd124af78..9a9b947386dd1fd196aaa25f4d4a8952a7c35b81#r74491369>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AH_jNrlhk74C5lXlP_q1SwAvrrGtA9Fnks5qe33ngaJpZM4Jf4Xo>
> .
>